src/Bridge/Symfony/Bundle/ErrorHandler/SymfonyExceptionHandler.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+declare(strict_types=1);
+/*
+ * @author     mfris
+ * @copyright  PIXELFEDERATION s.r.o.
+ * @license    Internal use only
+ */
+
+namespace K911\Swoole\Bridge\Symfony\Bundle\ErrorHandler;
+
+use K911\Swoole\Bridge\Symfony\HttpFoundation\RequestFactoryInterface;
+use K911\Swoole\Bridge\Symfony\HttpFoundation\ResponseProcessorInterface;
+use K911\Swoole\Server\RequestHandler\ExceptionHandler\ExceptionHandlerInterface;
+use ReflectionClass;
+use ReflectionMethod;
+use Swoole\Http\Request;
+use Swoole\Http\Response;
+use Symfony\Component\ErrorHandler\ErrorHandler;
+use Symfony\Component\HttpFoundation\Request as HttpFoundationRequest;
+use Symfony\Component\HttpKernel\HttpKernel;
+use Symfony\Component\HttpKernel\HttpKernelInterface;
+use Symfony\Component\HttpKernel\TerminableInterface;
+use Throwable;
+
+final class SymfonyExceptionHandler implements ExceptionHandlerInterface
+{
+    /**
+     * @var HttpKernelInterface
+     */
+    private $kernel;
+
+    /**
+     * @var RequestFactoryInterface
+     */
+    private $requestFactory;
+
+    /**
+     * @var ResponseProcessorInterface
+     */
+    private $responseProcessor;
+
+    /**
+     * @var ErrorHandler
+     */
+    private $errorHandler;
+
+    /**
+     * @var ReflectionMethod
+     */
+    private $throwableHandler;
+
+    public function __construct(
+        HttpKernelInterface $kernel,
+        RequestFactoryInterface $requestFactory,
+        ResponseProcessorInterface $responseProcessor,
+        ErrorHandler $errorHandler
+    ) {
+        $this->kernel = $kernel;
+        $this->requestFactory = $requestFactory;
+        $this->responseProcessor = $responseProcessor;
+        $this->errorHandler = $errorHandler;
+        $this->throwableHandler = $this->createThtowableHandler();
```

Comment:
Please use DI to handle this case too. You won't be able to write proper unit tests without it. Unit tests for this class are also necessary, because, among other things, it'll be easier to maintain this code.

Reply:
done